Library/Homebrew/cmd/test-bot.rb

```diff
@@ -651,7 +651,8 @@ def cleanup_before
         safe_system "git", "checkout", "-f", "master"
         # This will uninstall all formulae, as long as
         # HOMEBREW_REPOSITORY == HOMEBREW_PREFIX, which is true on the test bots
-        safe_system "git", "clean", "-ffdx", "--exclude=/Library/Taps/" unless ENV["HOMEBREW_RUBY"] == "1.8.7"
+        Tap.names.reject { |t| t == "homebrew/core" }.each { |s| safe_system "brew untap #{s}" }
+        safe_system "git", "clean", "-ffdx" unless ENV["HOMEBREW_RUBY"] == "1.8.7"
```

Comment:
Sorry if I'm missing the point, but omitting `"--exclude=/Library/Taps/"` here would mean that directory also gets wiped, right? This means that now all taps are gone at the start of the actual test run (including `homebrew/core`). Then why did we bother untapping them individually in the preceding line?